scripts/bump.sh

```diff
@@ -40,16 +39,18 @@ if [ "$RELEASE_CHANNEL" = "dev" ]; then
         .name = \"prisma-insider\" | \
         .displayName = \"Prisma - Insider\" | \
         .description = \"This is the Insider Build of the Prisma VSCode extension (only use it if you are also using the $(dev) version of the CLI.\" | \
+        .dependencies[\"@prisma/language-server\"] = \"$NEXT_EXTENSION_VERSION\" | \
         .preview = true" \
```

Comment:
When I try this locally with "dev" I get this weird error

![image](https://user-images.githubusercontent.com/746482/85741318-ef9af300-b71f-11ea-93b6-3c9af98c0337.png)